docs/README-AWS.md

```diff
@@ -106,97 +146,121 @@ Resources:
         ElasticServerlessForwarderCloudWatchLogsEvents: ""  ## FILL WITH A COMMA DELIMITED LIST OF CLOUDWATCH LOGS LOG GROUPS ARNS TO SET SUBSCRIPTION FILTERS ON THE LAMBDA FOR (IF ANY).
         ElasticServerlessForwarderS3Buckets: ""             ## FILL WITH A COMMA DELIMITED LIST OF S3 BUCKETS ARNS THAT ARE THE SOURCES OF THE S3 SQS EVENT NOTIFICATIONS (IF ANY).
 ```
-* Deploy the Lambda from SAR running the following command:
-  * ```commandline
+1. Deploy the Lambda function from SAR by running the following command:
+
+   ```commandline
     aws cloudformation deploy --template-file sar-application.yaml --stack-name esf-cloudformation-deployment --capabilities CAPABILITY_IAM CAPABILITY_AUTO_EXPAND
     ```
 
-#### Notes
-Due to a [bug](https://github.com/aws/serverless-application-model/issues/1320) in AWS CloudFormation, if you want to update the Events settings for the deployed Lambda, you will have to execute a deployment deleting the existing ones before actually apply the new updated ones.
+**Note:**
+Due to a [bug](https://github.com/aws/serverless-application-model/issues/1320) in AWS CloudFormation, if you want to update the Events settings for the deployment, you will have to manually delete existing settings before applying the new settings.
+
+### Deploy using Terraform
 
-### Terraform
-* Save the following yaml content as `sar-application.tf`
+1. Save the following yaml content as `sar-application.tf`
 ```
 provider "aws" {
   region = ""  ## FILL WITH THE AWS REGION WHERE YOU WANT TO DEPLOY THE ELASTIC SERVERLESS FORWARDER
 }
-
+```
+  ```
 data "aws_serverlessapplicationrepository_application" "esf_sar" {
   application_id = "arn:aws:serverlessrepo:eu-central-1:267093732750:applications/elastic-serverless-forwarder"
 }
 
-resource "aws_serverlessapplicationrepository_cloudformation_stack" "esf_cf_stak" {
+  ``` resource "aws_serverlessapplicationrepository_cloudformation_stack" "esf_cf_stak" {
```

Comment:
Is this supposed to be a new code block? Here's what it looks like now:

<img width="582" alt="image" src="https://user-images.githubusercontent.com/10479155/176271896-fb30f515-4652-47b5-93bd-aaf385e82936.png">